### PR TITLE
Further automate generating release notes

### PR DESCRIPTION
**Pull request by bess**
* By tagging PRs with Feature and Bug labels, we can automate marking them as such in the release notes.
* This will shorten and simplify the release process.
* Update README to reflect updated practice.

Connected to https://github.com/yalelibrary/YUL-DC/issues/296

### CODE REVIEW

**Comment by bess** [comment]
Screen Shot 2020-07-01 at 2 12 47 PM

File: README.md [hunk 1]
```diff
@@ -113,17 +113,10 @@ For the image instance to properly access images, you must also create a .secret
...
-5. Once those changes are merged to the `master` branch, in the github web UI go to `Releases` and tag a new release with the right version number. Paste in the release notes for this version from the changelog you generated. In the release notes, split out `Features`, `Bug Fixes`, and `Other`
-6. Once the CI build has completed for `master`, tag and push a docker hub image with the same release number:
-
-  ```
-  docker pull yalelibraryit/dc-blacklight:f93089f70e18d6c6d77ee1a6b3b4866b6d284078 <-- the tag created by circleci
-  docker tag yalelibraryit/dc-blacklight:f93089f70e18d6c6d77ee1a6b3b4866b6d284078 yalelibraryit/dc-blacklight:v1.2.1 <-- the new, semantically versioned tag
-  docker push yalelibraryit/dc-blacklight:v1.2.1 <-- now our semantically versioned tag will be available from docker hub
-  ```
-
+3. Go through all PRs since the last release and ensure that any **features** and **bug fixes** have been tagged with the appropriate label. No need to label it unless it is a feature or a bug fix.
```
Comment (alishaevn) [inline README.md:116]: I thought that adding a label to the PR indicating it as a feature or bug fix is the responsibility of whoever created the PR?

Can we add having that label as a part of our process that restricts PR's from being merged?

Comment (bess) [reply README.md:116]: @alishaevn Yes, it should be the responsibility of the person who made the PR, I guess I was just picturing that the person doing the release should check this. 
Also, the way I set things up it expects features and bug fixes to be tagged, and everything else will get classified under "Technical Enhancements"... we could instead require a label for each one, I guess?

Comment (alishaevn) [reply README.md:116]: @bess I like the idea of also having a "Technical Enhancements" label if it's one of the 3 headers for the change log.

Comment (alishaevn) [reply README.md:116]: I've added "technical enhancements" and "security fix" labels

**Pull request merged**